.travis.yml

```diff
@@ -19,14 +19,34 @@ notifications:
       - "chat.freenode.net#namecoin-dev"
     on_success: never
 
+jobs:
+  allow_failures:
+    - stage: lint
+  include:
+    - os: linux
+      stage: lint
+      language: go
+      go: "1.14.x"
+      script:
+        # lint stage
+        - go get -v -d -t ./...
+        - bash testdata/golangci-linter.bash
+
+stages:
+  - lint
+  - test
+
+# 'test' stage
 script:
-  - if [ "$TRAVIS_OS_NAME" = 'windows' ]; then bash testdata/travis.bash; fi
-  - go get -v -d -t ./...
-  - go test -v ./...
+  - env
+  - bash testdata/travis.bash
 
 env:
   global:
     # GITHUB_TOKEN for automatic releases
     - secure: "at1oJs7ib7glx3W+zk+OkT041LdknVXirIhN403CIihVUrlOhODY7yCTgvF4Rk0jYBJiT35Q2qxpgfWF2qGnsNsQmjG3ydDWQDCepDc/CgXfLyoiSTJK5vTK72dYWTVsBTycXbj1CbSy2X2ah/KWjc4RcgZ67ER7mDpRU5nFeow="
-    # Set this to the Go version to use for releases (must appear in version list above).
+      # Set this to the Go version to use for releases (must appear in version list above).
```

Comment:
Why is the whitespace added here?  AFAICT it flows better without the extra whitespace.